hack/registry.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
```

Comment:
Noting that this [duplicates](https://github.com/enterprise-contract/ec-cli/blob/f307f7afc70892a474a61d27ff29d3fbb35863b7/hack/test/kustomization.yaml#L24) the `hack/setup-[dev|test]-environment.sh`

Reply:
good catch. removed